python-sdk/example_dags/demo_parse_directory/transform.sql

```diff
@@ -1,5 +1,5 @@
 SELECT title, rating
 FROM {{ input_table }}
-WHERE genre1=='Animation'
```

Comment:
This syntax is correct for Postgres but incorrect for SQLite we use this file to run test which uses SQLite if you want to update you will have to update conn here too https://github.com/astronomer/astro-sdk/blob/main/python-sdk/example_dags/example_transform_file.py#L22

Reply:
@kelechi2020 @pankajastro would there be a way to represent this query that would work for both databases?
How should we move forward with this PR?